### DISCUSSION TITLE
Welcome to auth-module Discussions!

### DISCUSSION THREAD

**Discussion by pi0** [nuxt-community/auth-module#1426] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Intevel** [comment]
Big enhancement: https://github.com/nuxt-community/auth-module/issues/889

**Comment by Intevel** [comment]
@pi0 I think we should add Web3 things, like Metamask Auth or smth else.

**Reply by toniengelhardt** [reply]
A very interesting use case for crypto wallets could be to store a private key to encrypt and decrypt data of an app, aka. you open the app, download encrypted data from the server and use the key from the wallet for decryption. Not sure if such a flow is currently possible with e.g. MetaMask – this is just brainstorming – but something like that would be sick to avoid storing sensitive data (e.g. medical) on a server.

**Comment by Intevel** [comment]
Can we add a description to the github repository? @pi0 
Something like: 
The module authenticates users using a configurable authentication scheme or by using one of the directly supported providers. It provides an API for triggering authentication and accessing resulting user information.
